backend/accounts/admin.py

```diff
@@ -133,5 +172,23 @@ def get_groups(self, obj: User):
     get_groups.short_description = _("Groups")
 
 
+@admin.register(CommissionUser)
+class ComissionAdmin(UserAdmin):
+    """
+    User admin which only handles electoral commission users
+    """
+
+    # TODO: Use only relevant filters
+    # TODO: Move request_confirmations_deletion to be available only to this admin
+
+    def get_queryset(self, request) -> QuerySet:
+        return (
+            super()
+            .get_queryset(request)
+            .filter(groups__name__in=[COMMITTEE_GROUP, COMMITTEE_GROUP_READ_ONLY])
+            .exclude(groups__name__in=[STAFF_GROUP, SUPPORT_GROUP])
+        )
```

Comment:
Could we set an `objects = CommissionManager()` in the model and not use a `get_queryset()` here? 🤔